examples/javascript/js_example/templates/plain.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html' %}
+
+{% block intro %}
+  <a href="https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest"><code>XMLHttpRequest</code></a>
+  is the plain JavaScript way to make requests. It's natively supported
+  by all browsers.
+{% endblock %}
+
+{% block script %}
+  <script>
+    function addSubmit(ev) {
+      ev.preventDefault();
+      var request = new XMLHttpRequest();
+      request.addEventListener('load', addShow);
+      request.open('POST', '{{ url_for('add') }}');
```

Comment:
I'd use `{{ url_for('add')|tojson }}` instead of a quoted string. While not really relevant here it gives people a better example on how to properly pass data from Python to JS.